src/console_title.go

```diff
@@ -52,12 +45,23 @@ func (t *consoleTitle) getConsoleTitle() string {
 }
 
 func (t *consoleTitle) getTemplateText() string {
-	context := &TitleTemplateContext{
-		Root:   t.env.isRunningAsRoot(),
-		Path:   t.getPwd(),
-		Folder: base(t.getPwd(), t.env),
-		Shell:  t.env.getShellName(),
+	context := make(map[string]interface{})
+
+	context["Root"] = t.env.isRunningAsRoot()
+	context["Path"] = t.getPwd()
+	context["Folder"] = base(t.getPwd(), t.env)
+	context["Shell"] = t.env.getShellName()
+
+	// load environment variables into the map
+	envVars := os.Environ()
```

Comment:
If we do this, it needs to be part of environment. But hold your horses as I want to test a different approach first.